jit/src/instructions.rs

```diff
@@ -348,6 +348,22 @@ impl<'a, 'b> FunctionCompiler<'a, 'b> {
                             .push(JitValue::Bool(self.builder.ins().bint(types::I8, val)));
                         Ok(())
                     }
+                    (JitValue::Bool(a), JitValue::Bool(b)) => {
```

Comment:
python `bool` is a subclass of `int` and they can be mixed in operations. This pattern looks like matching only for (bool, bool) operands. Isn't this better to be combined to (int, int) case? Then it will allows also (bool, int) or (int, bool) operations.

Reply:
Thanks for response!

Yeah! makes sense , even in cpython that's how it is.
However the comparison methods for integers are not working for booleans.
Especially those signed versions. That's why I put it in different block.

I will make changes as per suggestion , and update it.

Reply:
Yeah! Makes sense , will update it. Thanks for response !